cypress/support/pages/home.pageObject.js

```diff
@@ -4,12 +4,42 @@ class HomePageObject extends PageObject {
   url = '/#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return cy.get(':nth-child(4) > .nav-link');
+  }
+
+  get logoOnHomePage() {
+    return cy.get('h1');
+  }
+
+  get signInLink() {
+    return cy.get(':nth-child(2) > .nav-link');
```

Comment:
The CSS selector for `signInLink` is also specific and could break if the HTML structure changes. Consider using a more robust selector, such as a data attribute, if available.